example-app/database/seeders/DatabaseSeeder.php

```diff
@@ -13,11 +18,20 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // User::factory(10)->create();
+        City::truncate();
+        Airline::truncate();
+        Flight::truncate();
+        City_Airline::truncate();
```

Comment:
no need for this, you can do a migrate fresh with a --seed

https://laravel.com/docs/11.x/migrations#drop-all-tables-migrate